tests/lib/openstack_query/handlers/test_client_side_handler_datetime.py

```diff
@@ -0,0 +1,358 @@
+import unittest
+from datetime import datetime
+from unittest.mock import patch
+from parameterized import parameterized
+
+from openstack_query.handlers.client_side_handler_datetime import (
+    ClientSideHandlerDateTime,
+)
+
+from tests.lib.openstack_query.mocks.mocked_props import MockProperties
+
+from enums.query.query_presets import QueryPresetsDateTime
+
+
+class ClientSideHandlerDateTimeTests(unittest.TestCase):
+    """
+    Run various tests to ensure that ClientSideHandlerDateTime class methods function expectedly
+    """
+
+    def setUp(self):
+        """
+        Setup for tests
+        """
+        super().setUp()
+        # sets filter function mappings so that PROP_1 is valid for all client_side
+        _FILTER_FUNCTION_MAPPINGS = {
+            preset: [MockProperties.PROP_1] for preset in QueryPresetsDateTime
+        }
+        self.instance = ClientSideHandlerDateTime(_FILTER_FUNCTION_MAPPINGS)
+
+    @parameterized.expand(
+        [(f"test {preset.name}", preset) for preset in QueryPresetsDateTime]
+    )
+    def test_check_supported_all_presets(self, name, preset):
+        """
+        Tests that handler supports all generic query client_side
+        """
+        self.assertTrue(self.instance.check_supported(preset, MockProperties.PROP_1))
+
+    @parameterized.expand(
+        [
```

Comment:
Can we extract this list out to be a class variable, as it should help the format mangling too